Brooklyn/Resources/Localization/Localizations.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Localizations.swift
+//  Brooklyn
+//
+//  Created by Peter Bødskov on 06/10/2019.
+//  Copyright © 2019 Pedro Carrasco. All rights reserved.
+//
+
+import Foundation
+
+struct Localizations {
+    enum Preferences {
+        static let version = "Version:"
```

Comment:
I would drop the `:`.`Version 2.0.1` looks better than `Version: 2.0.1` **IMHO**. But leave it as you prefer!

Reply:
No worries, that's an easy fix and I actually agree with you, don't know why I added that `:`  in the first place🤷‍♂